running-coreos/platforms/libvirt/index.md

```diff
@@ -215,3 +164,78 @@ ssh coreos0
 
 Now that you have a machine booted it is time to play around.
 Check out the [CoreOS Quickstart]({{site.baseurl}}/docs/quickstart) guide or dig into [more specific topics]({{site.baseurl}}/docs).
+
+# Running CoreOS cluster demo on libvirt
+
+This guide explains how to run three-nodes demo CoreOS cluster with libvirt.
+
+## Bash Script
+
+Save following `deploy_coreos_cluster.sh` script into your host filesystem:
+
+```sh
+wget http://stable.release.core-os.net/amd64-usr/current/deploy_coreos_cluster.sh
```

Comment:
I think this would be `https://github.com/coreos/docs/blob/master/running-coreos/platforms/libvirt/deploy_coreos_cluster.sh` or `https://raw.githubusercontent.com/coreos/docs/master/running-coreos/platforms/libvirt/deploy_coreos_cluster.sh`